src/main/java/net/openhft/chronicle/queue/impl/single/TSQueueLock.java

```diff
@@ -122,15 +121,25 @@ public void waitForLock() {
         }
     }
 
+    private void warnAndForceUnlock(String msg, long value) {
+        warnLock(msg, value);
```

Comment:
duplicated but not quite the same code - we should address with https://github.com/OpenHFT/Chronicle-Queue/issues/994